lib/cli.ts

```diff
@@ -199,7 +213,7 @@ function infer(yargs: Argv): Stream {
 }
 
 yargs
-	.usage(i18n('Usage: $0 <command> [files...] [options]'))
+	.usage(i18n('Usage: $0 <command> [files,dirs...] [options]'))
```

Comment:
Hmm it might be that some users are not familiar with the term 'glob' so I thought to keep it simple.